functionalTest/test/utilities/reusableLibrary.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Created this file to include all reusable methodes in future.
+ */
+class reusableLibrary {}
```

Comment:
Not a fan of creating classes/files 'just in case', but letting this one slide :) If it turns out it isn't used in a week or two, we'll bin it

Reply:
Sure... Will remove it if not used in a week or two.